logging.conf

```diff
@@ -0,0 +1,46 @@
+# Note Expects 'logDir' passed in with location for file output.
+
+#============================ formatters ======================================
+
+[formatters]
+keys=std,console
+
+[formatter_std]
+format=%(asctime)s [%(name)s:%(levelname)s] %(message)s
+datefmt=
+
+[formatter_console]
+format=%(asctime)s %(levelname)s %(message)s
+datefmt=%H:%M:%S
+
+#============================ handlers ========================================
+
+[handlers]
+keys=std,console
+
+[handler_std]
+class=handlers.RotatingFileHandler
+# args: filename, open mode, max file size, backup file count
+args=('solmanager.log', 'a', 20000000, 5)
+formatter=std
+
+[handler_console]
+class=StreamHandler
+args=()
+formatter=console
+
+#============================ loggers =========================================
+
+[loggers]
+keys=root,solmanager
+
+[logger_root]
```

Comment:
please add comment about what this is. I believe `root` refers to the default one?